src/bindings/python/src/pyopenvino/core/core.cpp

```diff
@@ -110,6 +110,21 @@ void regclass_Core(py::module m) {
             :rtype: object
         )");
 
+    cls.def(
+        "get_property",
```

Comment:
Please add test cases for it for Python API.

Reply:
Add python test for get_property without parameter device name.
Test usage:
$ python -m pytest ../src/bindings/python/tests/test_runtime/test_properties.py